test-integration/filter-lido.js

```diff
@@ -157,39 +56,28 @@ contract("Lido Filter", (accounts) => {
   describe("Selling via CurvePool", () => {
     beforeEach(async () => {
       // Stake some funds to use to test selling
-      const data = lido.contract.methods.submit(accounts[5]).encodeABI();
-      const transaction = encodeTransaction(lido.address, 100, data);
-
-      await manager.relay(
-        module,
-        "multiCall",
-        [wallet.address, [transaction]],
-        wallet,
-        [owner],
-        1,
-        ETH_TOKEN,
-        relayer);
+      await argent.multiCall(wallet, [
+        [lido, "submit", [accounts[5]], 100]
+      ]);
     });
 
     it("should allow selling stETH via Curve", async () => {
-      const transactions = [];
       let data = lido.contract.methods.approve(curve.address, 99).encodeABI();
-      let transaction = encodeTransaction(lido.address, 0, data);
-      transactions.push(transaction);
+      const lidoTransaction = utils.encodeTransaction(lido.address, 0, data);
       data = curve.contract.methods.exchange(1, 0, 99, 1).encodeABI();
-      transaction = encodeTransaction(curve.address, 0, data);
-      transactions.push(transaction);
+      const curveTransaction = utils.encodeTransaction(curve.address, 0, data);
+      const transactions = [lidoTransaction, curveTransaction];
 
       const before = await utils.getBalance(wallet.address);
-      const txReceipt = await manager.relay(
-        module,
+      const txReceipt = await argent.manager.relay(
```

Comment:
could we use `argent.multiCall` here instead?

Reply:
Done